README.md

```diff
@@ -28,11 +18,19 @@ the source code given only:
   - or run ./prepare_dependencies.py cmake
```

Comment:
`./prepare_dependencies` will install cmake without the cmake parameter. Not sure if this is necessary.